src/layers/icon-layer/icon-layer.js

```diff
@@ -20,27 +20,15 @@
 
 import Layer from '../base-layer';
 import memoize from 'lodash.memoize';
+import window from 'global/window';
+
 import {hexToRgb} from 'utils/color-utils';
-import {svgIcons as SvgIcons} from './svg-icons.json';
 import SvgIconLayer from 'deckgl-layers/svg-icon-layer/svg-icon-layer';
 import IconLayerIcon from './icon-layer-icon';
-import {ICON_FIELDS} from 'constants/default-settings';
+import {ICON_FIELDS, CLOUDFRONT} from 'constants/default-settings';
 import IconInfoModalFactory from './icon-info-modal';
 
-const IconInfoModal = IconInfoModalFactory();
-const IconIds = SvgIcons.map(d => d.id);
-const SvgIconGeometry = SvgIcons.reduce(
-  (accu, curr) => ({
-    ...accu,
-    [curr.id]: curr.mesh.cells.reduce((prev, cell) => {
-      cell.forEach(p => {
-        Array.prototype.push.apply(prev, curr.mesh.positions[p]);
-      });
-      return prev;
-    }, [])
-  }),
-  {}
-);
+export const SVG_ICON_URL = `${CLOUDFRONT}/icons/svg-icons.json`;
```

Comment:
are we sure we want to create this connection between static assets and cloudfront bucket?
This will create a direct dependency with the specific cloudfront url and our assets, if someone removes the icon file from cloudfront it may break kepler.gl instances.

What about using kepelr.gl github repo as an alternative solution?

Reply:
We are already loading several assets, e.g. layer icons, logos, map styles from the same cloudfront, this doesn't add a connection. Just following the old practice